ansible/roles/probe-services/templates/tor_targets.json

```diff
@@ -1,124 +1,101 @@
 {
```

Comment:
I don't know the answer to this. I updated both because I wasn't sure which one is actually used by the backend to provide data to probes.
@FedericoCeratto should know the answer to this.

If one of the two roles is deprecated, I think we should delete it.

Reply:
The "probe-services" role is legacy and was used for the backend hosts on Greenhost like hkg-ps.ooni.nu 
The new backend uses "ooni-backend"